### PR TITLE
New boiler and fan entities

### PR DESCRIPTION
**Pull request by annieDehghani**
Added a new boiler type. 
Added a new fan type.
Moved fields on to opt_uses  for generator, ATS and circ pump.

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/FAN.yaml [hunk 1]
```diff
@@ -36,7 +36,15 @@ FAN_SS_VSC:
   - FAN
   - SS
   - VSC
-
+  
+FAN_VSC:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/FAN.yaml:40]: This should not be needed, as `FAN_SS_VSC` exists. Please use that.

**Review by tasodorff — CHANGES_REQUESTED** [review]
We cannot move fields from USES to OPT_USES. New types are required, but its probably more appropriate in your deployment to use MISSING fields in the BC translation, if they are truly missing.

**Review by tasodorff — CHANGES_REQUESTED** [review]
We cannot move fields from USES to OPT_USES. New types are required, but its probably more appropriate in your deployment to use MISSING fields in the BC translation, if they are truly missing.

**Comment by annieDehghani** [comment]
@tasodorff thank you for that feedback. I was not aware of the option to use MISSING in the translation. Thank you for pointing me to that.

**Pull request closed without merging**